diff --git a/node_modules/react-native-tag-input/index.js b/node_modules/react-native-tag-input/index.js
index 9897fa2..db196eb 100644
--- a/node_modules/react-native-tag-input/index.js
+++ b/node_modules/react-native-tag-input/index.js
@@ -15,10 +15,10 @@ import {
   Dimensions,
   TouchableWithoutFeedback,
   ScrollView,
-  ViewPropTypes,
   Platform,
 } from 'react-native';
 import invariant from 'invariant';
+import {ViewPropTypes, TextPropTypes} from 'deprecated-react-native-prop-types'; 
 
 const windowWidth = Dimensions.get('window').width;
 
@@ -115,7 +115,7 @@ class TagInput<T> extends React.PureComponent<Props<T>, State> {
     tagColor: PropTypes.string,
     tagTextColor: PropTypes.string,
     tagContainerStyle: ViewPropTypes.style,
-    tagTextStyle: Text.propTypes.style,
+    tagTextStyle: TextPropTypes.style,
     inputDefaultWidth: PropTypes.number,
     inputColor: PropTypes.string,
     inputProps: PropTypes.shape(TextInput.propTypes),
@@ -378,7 +378,7 @@ class Tag extends React.PureComponent<TagProps> {
     tagColor: PropTypes.string.isRequired,
     tagTextColor: PropTypes.string.isRequired,
     tagContainerStyle: ViewPropTypes.style,
-    tagTextStyle: Text.propTypes.style,
+    tagTextStyle: TextPropTypes.style,
   };
   curPos: ?number = null;
 
